src/layouts/BaseLayout.astro

```diff
@@ -49,12 +53,12 @@ const navigation = Navigation.create({ nav: sideNavConfig, content });
     <meta name="viewport" content="width=device-width">
     <title>{metaTitle}</title>
     <meta name="description" content={description}>
-    <meta property="img" content={`${Astro.url.origin}${img}`}>
+    <meta property="img" content={img}>
```

Comment:
Question: Was this change intended?

Reply:
@MeganSteenkamp Yep - converting these to relative links avoids complexity when switching between environments